docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -66,34 +66,38 @@ For example, the simple Iou in :ref:`simple_iou` allowed the no-op where the ``o
+**Record time (RT)** is the time assigned by the persistence layer. It represents the time that the transaction is “physically” recorded. For example, “The backing database ledger has assigned the timestamp of such-and-such time to this transaction.” For a Daml transaction to be valid, it must have an LT and RT that are not too different from each other. This is the only purpose in the RT - to ensure that transactions are being recorded in a timely manner. The RT should  roughly align with the LT. This is the requirement known as “bounded skew.”
 
-``getTime`` is an action that gets the LT from the ledger. In the above example, that time is taken apart into day of week and hour of day using standard library functions from ``DA.Date`` and ``DA.Time``. The hour of the day is checked to be in the range from 8 to 18.
+Each Daml ledger has a policy on the allowed difference between LT and RT called the *skew*. If it's too far off, the transaction will be rejected. Consequently, the participant keeps a running estimate of the time required to record a transation. The participant assigns the LT based on that estimate.
 
-Consider the following example: Suppose that the ledger had a skew of 10 seconds. At 17:59:55, Alice submits a transaction to redeem an Iou. One second later, the transaction is assigned a LT of 17:59:56, but then takes 10 seconds to commit and is recorded on the ledger at 18:00:06. Even though it was committed after business hours, it would be a valid transaction and be committed successfully as ``getTime`` will return 17:59:56 so ``hrs == 17``. Since the RT is 18:00:06, ``LT - RT <= 10 seconds`` and the transaction won't be rejected.
+The RT is not really relevant outside the ledger. The reason for this is that ``getTime`` returns LT and is the logical time of the transaction. The primary role of RT from the application's perspective is to set an upper-bound on the amount of skew between the logical business concept of time and the wall-clock time. A consistent zero-skew is not feasible given this is a distributed system.
 
-Time therefore has to be considered slightly fuzzy in Daml, with the fuzziness depending on the skew parameter.
+Returning to the theme of *business hours*, consider the following example: Suppose that the ledger had a skew of 10 seconds. At 17:59:55 just before the end of business hours, Alice submits a transaction to redeem an Iou. One second later, the transaction is assigned a LT of 17:59:56, but then takes 10 seconds to commit. The transaction is recorded in the underlying backing store at 18:00:06, *after business hours*. Even though it was committed after business hours, it would be a valid transaction and be committed successfully as ``getTime`` will return 17:59:56. Since the RT is 18:00:06, ``LT - RT <= 10 seconds`` and the transaction won't be rejected.
```

Comment:
This language is misleading. Talking about the the RT being the "commit" time is confusing. Yes, the RT was the time it was recorded/persisted, but to talk about that as a "commit-time" is to import RDBMS language into Daml that doesn't apply here.

The ledger time _is_ the time the transaction ran. The record time _is_ the time the ledger recorded/persisted that fact. Of these the LT is in fact the closest to what an RDBMS means by a "commit" time.

The language used above suggests that the Daml platform has violated the business rules in some way—which is manifestly not true.

Reply:
Fixed with d56fa7edc54e721e9e0df4c009defc457f0eddd7